src/utilities/theme/tokens.ts

```diff
@@ -10,6 +10,8 @@ const Shadow = {
     '-1px 0px 20px var(--p-shadow-from-ambient-light), 0px 1px 5px var(--p-shadow-from-direct-light)',
   modalShadow:
     '0px 6px 32px var(--p-shadow-from-ambient-light), 0px 1px 6px var(--p-shadow-from-direct-light)',
+  tooltipShadow:
+    '-1px 0px 20px var(--p-shadow-from-ambient-light), 0px 1px 5px var(--p-shadow-from-direct-light)',
```

Comment:
Note that this is the same value as `popoverShadow`.

Reply:
This is fine for now, but let’s think of a better naming convention for this.

Reply:
I'm not crazy about seeing another token declared from this that is identical to an existing token. Can we just re-use the existing one with the aim of following up later with a better naming convention?

Reply:
For sure! Made a note to follow up on the shadow variables in this issue.